features/finance/automatic_billing_postpaid.feature

```diff
@@ -0,0 +1,121 @@
+@stats
+Feature: Automatic billing with plan changes on POSTPAID
+  As a provider I want to differentiate costs added by the automatic
+  billing job and my manually created invoices
+
+  Background:
+    Given a provider with billing and finance enabled
+    Given the provider service allows to change application plan directly
+    And the provider has one buyer
+    And the provider has a paid application plan "Paid" of 31 per month
+    And the provider has another paid application plan "Expensive" of 3100 per month
+
+  Scenario: Monthly fee on application plan upgrading the same day
+    Given all the rolling updates features are off
+    And the date is 1st January 2017
+    And the buyer signed up for plan "Paid"
+    Then the buyer changed to plan "Expensive"
+    When time flies to 3rd February 2017
+
+    Then the buyer should have following line items for "January, 2017" invoice:
+      | name                        | quantity |  cost     |
+      | Fixed fee ('Paid')          |          |    31.00  |
+      | Refund ('Paid')             |          |   -31.00  |
+      | Fixed fee ('Expensive')     |          | 3,100.00  |
+      | Total cost                  |          | 3,100.00  |
+
+  Scenario: Monthly fee on application plan downgrading the same day
+    Given all the rolling updates features are off
+    And the date is 1st January 2017
```

Comment:
Seems like `Given all the rolling updates features are off
     And the date is 1st January 2017` are valid for all scenario's and could be moved to Background?